src/input/InputDriverManager.cc

```diff
@@ -129,6 +129,18 @@ std::string InputDriverManager::listDrivers()
     return stream.str();
 }
 
+void InputDriverManager::closeDrivers()
+{
+	stopRequest = true;
+	timer->stop();
+	InputEventMapper::instance()->stop();
+
+    for (drivers_t::iterator it = drivers.begin();it != drivers.end();it++) {
```

Comment:
C++11 range iterators could be used here: `for (auto driver : drivers) {`.

Also: indentation mismatch (in multiple places in this PR)

Reply:
changed for loops to c++11 for loop
indentation matched to preexisting code